db/migrate/20210610090328_add_uplift_to_early_career_teacher_profile.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class AddUpliftToEarlyCareerTeacherProfile < ActiveRecord::Migration[6.1]
+  def change
+    change_table(:early_career_teacher_profiles, bulk: true) do |table|
+      table.column :sparsity_uplift, :boolean, null: false, default: false
+      table.column :pupil_premium_uplift, :boolean, null: false, default: false
```

Comment:
How are we planning to populate this stuff?

Reply:
It's in this commit 42ef1cc

Reply:
Well, some of it is. What about the profiles that don't exist yet, but will exist in a few days or weeks, when the ECTs are added by induction coordinators?

I imagine you want to add something adding it to `save!` method of `AddParticipantForm`.

Reply:
yeah, Paul is working on this at the moment actually, it's exactly there or in the model callback

Reply:
Also, perhaps a more important question. If we store this data against the school, are we sure we want to duplicate it on the ECT profiles? 
Are we doing it for efficiency reason? Are we doing it because when user switches school we want to remember it? Are we doing it because we are worried that over time this stuff will change?

Reply:
> Also, perhaps a more important question. If we store this data against the school, are we sure we want to duplicate it on the ECT profiles?
> Are we doing it for efficiency reasons? Are we doing it because when user switches school we want to remember it? Are we doing it because we are worried that over time this stuff will change?

"Are we doing it because when user switches school we want to remember it?" - Yes, because they can in theory change schools between the point of registration and (months later) when we calculate the payment. We suspect these school changes may be done via support initially. The business rules we were given are that it is the uplift at the time they are registered that matters, not the uplift at the point we calculate.